src/main/java/Duke/Tasks/EventTask.java

```diff
@@ -0,0 +1,24 @@
+package Duke.Tasks;
+
+import Duke.*;
+
+public class EventTask extends Task {
+    private String event;
+    private String from;
+    private String to;
+
+    public EventTask(String task) {
+        super(task);
+        String[] splitsy =  task.split("/");
+        event = splitsy[0];
+        from = splitsy[1].split(" ", 2)[1];
+        to = splitsy[2].split(" ", 2)[1];
+    }
+
+
+    public String toString() {
+        return String.format("[E] | %s | %s | %s - %s",
+                this.isDone() ? "[X]" : "[ ]",
+                event, Parser.dateToString(from.trim()), Parser.dateToString(to.trim()));
```

Comment:
Will it be better if the comma follows the next line in a line break. i.e. ```, this.isDone() ? "[X]" : "[ ]"``` for the second line?